recipes/platform.equality/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.8)
+project(test_package)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+add_executable(${PROJECT_NAME} test_package.cpp)
+target_link_libraries(${PROJECT_NAME} CONAN_PKG::gtest)
+target_link_libraries(${PROJECT_NAME} CONAN_PKG::platform.equality)
```

We tend to not use targets in test packages for uniformity. 
Unless we're testing cmake_find_package modules, which we are not in this case.
```suggestion
conan_basic_setup()

add_executable(${PROJECT_NAME} test_package.cpp)
target_link_libraries(${PROJECT_NAME} ${CONAN_LIBS})
```

---

Ok, thanks